docs/reference/operator_parameters.md

```diff
@@ -443,4 +443,22 @@ scalyr sidecar. In the CRD-based configuration they are grouped under the
   Memory limit value for the Scalyr sidecar. The default is `1Gi`.
 
 
+## Logical backup
+
+  These parameters configure a k8s cron job managed by the operator to produce Postgres logical backups.
+  In the CRD-based configuration those parameters are grouped under the `logical_backup` key.
+
+  * **enable_logical_backup**
+    Determines if the operator should initiate the backup of all Postgres clusters it controls. Default: false.
```

Comment:
I think it's formulated incorrectly, operator doesn't initiate the backup itself, it creates a job for that.